### ISSUE TITLE
WCAG 1.1.1 Missing alt text on illustrations

### ISSUE THREAD

**Issue by arnfinn** [mong/mongts#1084]
Observations:

Missing alt text on illustrations, examples:

Illustration of persons: https://www.helsenorge.no/pasientjournal

Diagram "hospital referral areas" https://www.skde.no/helseatlas/en/v2/kronikere/
